### ISSUE TITLE
Requested module 'fast-deep-equal' does not provide an export named 'default'

### ISSUE THREAD

**Issue by dmoyadev** [fawmi/vue-google-maps#152]
**Describe the bug**
With version 0.9.79, there is something broken related to default exports, as I'm receiving this message in the console that breaks everything in my app:
`The requested module '/node_modules/fast-deep-equal/index.js?v=dfdb27d1' does not provide an export named 'default'
`

It is fixed once I downgraded to 0.9.67

**Comment by NathanAP** [comment]
Same here, I'm getting this error after making a `npm update` to 0.9.79

**Comment by joshhammer** [comment]
Same here. Can't get it to work anymore. Not even with downgrading. Did that work for anybody else? How did you downgrade exactly?

**Comment by NathanAP** [comment]
> Same here. Can't get it to work anymore. Not even with downgrading. Did that work for anybody else? How did you downgrade exactly?

Downgrade to `v0.9.67`, you just need to set the exact version in your `package.json`, delete the `node_modules` directory and run `npm install` again.

**Comment by mrc-bsllt** [comment]
It does not even work for me with version v0.9.67 (Nuxt 3 3.0.0-rc.11). I also tried adding @googlemaps/markerclusterer in the nuxt.config.ts transpile.

nuxt.config.ts
```
build: {
  extractCSS: true,
  transpile: ['swiper', 'shipit-cli', '@fawmi/vue-google-maps', '@googlemaps/markerclusterer']
}
```
Does anyone know how to do this?

**EDIT:**
this problem only exists locally, in production everything works fine!

**Comment by wasitum** [comment]
I have the same issue with Nuxt 3.0.0-rc.10 & package v0.9.79. Tried to downgrade to 0.9.67 but same issue. As @mrc-bsllt mentioned, this works in production but not locally. Any lead on that ?

**Comment by zakariamehbi** [comment]
> I have the same issue with Nuxt 3.0.0-rc.10 & package v0.9.79. Tried to downgrade to 0.9.67 but same issue. As @mrc-bsllt mentioned, this works in production but not locally. Any lead on that ?

Try fixing the version, this way : `"@fawmi/vue-google-maps": "0.9.67"`

**Comment by wasitum** [comment]
> > I have the same issue with Nuxt 3.0.0-rc.10 & package v0.9.79. Tried to downgrade to 0.9.67 but same issue. As @mrc-bsllt mentioned, this works in production but not locally. Any lead on that ?
> 
> Try fixing the version, this way : `"@fawmi/vue-google-maps": "0.9.67"`

This process worked : 
- removed node_modules and the yarn lock file
- changed the version to what you said in the package.json file
- installed the package and ran the site again

Thanks !

**Comment by reslear** [comment]
hi @fawmi what is the package maintenance status?

**Comment by NathanAP** [comment]
Really hope that Fawmi keeps this maintained, this package is the best to manipulate maps I found so far.

**Comment by mrc-bsllt** [comment]
> > > I have the same issue with Nuxt 3.0.0-rc.10 & package v0.9.79. Tried to downgrade to 0.9.67 but same issue. As @mrc-bsllt mentioned, this works in production but not locally. Any lead on that ?
> > 
> > 
> > Try fixing the version, this way : `"@fawmi/vue-google-maps": "0.9.67"`
> 
> This process worked :
> 
> * removed node_modules and the yarn lock file
> * changed the version to what you said in the package.json file
> * installed the package and ran the site again
> 
> Thanks !

My problem was that I had not removed '^' from the package version.
I changed this "@fawmi/vue-google-maps": "^0.9.67" for this "@fawmi/vue-google-maps": "0.9.67", and it works!

**Comment by tahaipek** [comment]
Can you try this solution https://github.com/fawmi/vue-google-maps/issues/148#issuecomment-1235143844

**Comment by reslear** [comment]
@tahaipek with vite 3 don't work

**Comment by tahaipek** [comment]
I'm using Vite 3.1.3. This solution worked for me.

**Comment by Giuseppetm** [comment]
Same problem here..

**Comment by Giuseppetm** [comment]
> Can you try this solution [#148 (comment)](https://github.com/fawmi/vue-google-maps/issues/148#issuecomment-1235143844)

I can confirm this fixes the problem, thank you.

**Comment by reslear** [comment]
Is it possible to sponsor the development somehow?

**Comment by NathanAP** [comment]
> I'm using Vite 3.1.3. This solution worked for me.

Its working in `v3.1.3` for me as well, thank you so much! 

Is it confirmed that it is this the expected behavior? I believe we need a confirmation by @fawmi that this repo is still being maintaned. This was the best Google Maps repo I found and it would be awesome if you guys keep it up with it.

**Comment by tahaipek** [comment]
I agree with you too, @NathanAP . I hope @fawmi continues to developed this package.

**Comment by gbrian** [comment]
> Can you try this solution [#148 (comment)](https://github.com/fawmi/vue-google-maps/issues/148#issuecomment-1235143844)

Worked for me, thanks.

**Comment by rafidgotit** [comment]
> 

Thank you so much for the solution. Worked for me.

**Comment by josuelrocha** [comment]
> > Can you try this solution [#148 (comment)](https://github.com/fawmi/vue-google-maps/issues/148#issuecomment-1235143844)
> 
> Worked for me, thanks.

perfect, it worked for me too.

**Comment by WaheedNodesol** [comment]
I have the same issue with Quasar + Vite + vue3

Try fixing the version, this way : "@fawmi/vue-google-maps": "0.9.67"

This process worked :

removed node_modules and the npm lock file,
changed the version to what you said in the package.json file,
installed the package and run the site again
Thanks!

**Comment by fenghsu** [comment]
Thank you. Worked for me :)

Follow below process
1. npm uninstall @fawmi/vue-google-maps
2. npm install -S @fawmi/vue-google-maps@0.9.67

**Comment by NathanAP** [comment]
Guys, please read https://github.com/fawmi/vue-google-maps/issues/159

**Comment by bchewy** [comment]
> #159

This worked for me as well. 

Thanks!

**Comment by mydaycare-carefree** [comment]
Does not work with Vite 4.2.0

**Comment by Attar74** [comment]
![Screenshot from 2024-04-27 03-24-22](https://github.com/fawmi/vue-google-maps/assets/25443951/b568f57f-12d1-4281-ad9e-5893ccd046dc)
> > > I have the same issue with Nuxt 3.0.0-rc.10 & package v0.9.79. Tried to downgrade to 0.9.67 but same issue. As @mrc-bsllt mentioned, this works in production but not locally. Any lead on that ?
> > 
> > 
> > Try fixing the version, this way : `"@fawmi/vue-google-maps": "0.9.67"`
> 
> This process worked :
> 
> * removed node_modules and the yarn lock file
> * changed the version to what you said in the package.json file
> * installed the package and ran the site again
> 
> Thanks !

This worked for me but the map details are hidden give the attached SS a look 
 "@fawmi/vue-google-maps": "0.9.67",
 "nuxt": "^3.11.2",

**Comment by RilindArifi** [comment]
vite.config.js

```
optimizeDeps: {
        include: [
            "@fawmi/vue-google-maps",
            "fast-deep-equal",
        ],
    },
```

**Comment by nbyloff** [comment]
> I'm using Vite 3.1.3. This solution worked for me.

Years later I am using vite ^6.2.0 and this solution still works. However, it appears this package is abandoned?

**Comment by julngomz** [comment]
Build fails with docker:

<img width="1271" alt="Image" src="https://github.com/user-attachments/assets/2eb2ef20-82c5-4a63-b83d-f60a8b79bc82" />
